packages/assets-controllers/CHANGELOG.md

```diff
@@ -7,6 +7,15 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [53.2.0]
+
+### Changed
```

Comment:
This is probably only "fixes" no? So we could change it to a "Fixed" instead? (and we should change the version to be 53.1.1 probably?)